CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - [BUGFIX] Operator: Fix MetricsInstance Service port (@hjet)
 
+- [BUGFIX] Add relabel_config.libsonnet to logs templates dir (@hjet)
```

Comment:
Can this be changed to say what user behavior got fixed? ie mention it's for the operator and how relabel rules for podlogs weren't working